src/CI3Compatible/Database/CI_DB_query_builder.php

```diff
@@ -358,6 +362,25 @@ public function order_by(
         return $this;
     }
 
+    /**
+     * GROUP BY
+     *
+     * @param mixed $group_by what field or fields to group_by
+     *
+     * @return CI_DB_query_builder
+     */
+    public function group_by(mixed $group_by): self
```

Comment:
Please use the same method signature.
https://github.com/bcit-ci/CodeIgniter/blob/45576ef6e62b5ff59da67f1697ee8c57809c7719/system/database/DB_query_builder.php#L1120-L1127